lib/Route.js

```diff
@@ -31,7 +31,7 @@ Route.createRoute = function (patternConfig, schema, handlers, openApiDocs) {
     // Validate the schema
     var isSchemaValid = validateSchema(schema);
     if (!isSchemaValid) {
-        throw new Error("Invalid schema");
+        throw new Error("Invalid schema" + JSON.stringify(schema)); // TODO revert
```

Comment:
This is to help debugging for now?

Reply:
Reverted, I was looking into the failure from updating the jsen patch version